packages/plugins/jwt-auth/src/index.ts

```diff
@@ -0,0 +1,81 @@
+import { type Plugin as YogaPlugin } from 'graphql-yoga';
+import type { MeshServePlugin } from '@graphql-mesh/serve-runtime';
+import {
+  useJWT as useYogaJWT,
+  type JWTExtendContextFields,
+  type JwtPluginOptions,
+} from '@graphql-yoga/plugin-jwt';
+
+export {
+  createInlineSigningKeyProvider,
+  createRemoteJwksSigningKeyProvider,
+  extractFromCookie,
+  extractFromHeader,
+  type GetSigningKeyFunction,
+  type JWTExtendContextFields,
+  type JwtPluginOptions,
+  type ExtractTokenFunction,
+} from '@graphql-yoga/plugin-jwt';
+
+export type JWTAuthPluginOptions = Omit<JwtPluginOptions, 'extendContext'> & {
+  forward?: {
+    claims?: boolean | string;
+    token?: boolean | string;
+    extensionsFieldName?: string;
+  };
+};
+
+export function useExtractedJWT(config: {
```

Comment:
Not sure about this name. What do you think about `usePropagatedJWT` or `useForwardedJWT` something among the line ? To better imply the fact the JWT that will be extracted is coming from the gateway ?